build.gradle

```diff
@@ -70,16 +70,16 @@ ext {
 
     testing = [
             junit             : 'junit:junit:4.12',
-            robolectric       : 'org.robolectric:robolectric:3.2.2',
-            mockito           : 'org.mockito:mockito-core:2.7.7',
+            robolectric       : 'org.robolectric:robolectric:3.3.2',
+            mockito           : 'org.mockito:mockito-core:2.8.9',
             privateConstructor: 'com.pushtorefresh.java-private-constructor-checker:checker:1.2.0',
-            assertJ           : 'org.assertj:assertj-core:3.6.2'
+            assertJ           : 'org.assertj:assertj-core:3.8.0'
     ]
 
     javaVersion = JavaVersion.VERSION_1_7
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '3.3'
+    gradleVersion = '3.5'
     distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-all.zip"
```

Comment:
when we are anyways changing everything this line can be changed to `distributionType = Wrapper.DistributionType.ALL`